tests/features/user/profile.feature

```diff
@@ -58,8 +58,12 @@ Feature: User profile
     # A user should not be able to edit the profile page of another user.
     When I go to the public profile of "Domenico Ghirlandaio"
     Then I should not see the link "Edit"
+    # Verify that the user's "Country of origin" field is visible on its profile.
+    When I go to the public profile of "Leonardo Da Vinci"
+    Then I should see the text "Country of origin:" in the "Header" region
+    And I should see the link "Italy" in the "Header" region
```

Comment:
In the screenshot attached in the roiginal ticket, this is not a link. Plus, we do not have anything styled for the taxonomy pages. Please, change this into the "featured" view mode of the user to not link to the original entity.

Reply:
Alright!
I also adjusted the tests.